src/main.py

```diff
@@ -40,34 +40,28 @@
 
 def dynatrace_gcp_extension(event, context):
     """
-    Starting point for installation as a GCP function. See https://cloud.google.com/functions/docs/calling/pubsub#event_structure
+    Starting point for installation as a GCP function.
+    See https://cloud.google.com/functions/docs/calling/pubsub#event_structure
     """
     try:
-        asyncio.run(handle_event(event, context))
+        asyncio.run(query_metrics(None, None))
     except Exception as e:
         traceback.print_exc()
         raise e
 
 
-async def async_dynatrace_gcp_extension(project_ids: Optional[List[str]] = None, services: Optional[List[GCPService]] = None):
+async def async_dynatrace_gcp_extension(services: Optional[List[GCPService]] = None):
     """
-    Used in docker or for tests
+    Starting point for installation as an autopilot cluster and for tests.
```

Comment:
1 I think this code is not limited to autopilot, you can have your existing GKE
2 This code is not really k8s specific, let's focus on the fact that it is for "long-running" or "container"

Reply:
Right. I'll change the comment later too